src/vrapp/VolumeVisualizationApp.cpp

```diff
@@ -542,42 +539,40 @@ std::cout << "init vizapp " << std::endl;
     glClearDepth(1.0f);
     glDepthFunc(GL_LEQUAL);
     glClearColor(0.0, 0.0, 0.0, 1);
-     std::cout << "init vizapp end" << std::endl;
+    std::cout << "init vizapp end" << std::endl;
+    const float duration = 10.5f;
```

Comment:
which duration is this?

Reply:
None. It is old code from initial tests with Choreograph. I removed it.